src/libs/ck-libs/ampi/ddt.h

```diff
@@ -43,6 +44,7 @@
 #define CkDDT_INDEXED             47
 #define CkDDT_HINDEXED            48
 #define CkDDT_STRUCT              49
+#define CkDDT_MAX_PREDEFINED_TYPE 50
```

Comment:
This name seems confusing to me with AMPI_PMAX_PREDEFINED_TYPE defined above. I think something like CkDDT_MAX_USERDEFINED_TYPE would be better. Could also rename all these CkDDT_* macros to AMPI_*

Reply:
Renamed